### PR TITLE
Proposal to add a README file.

### PR DESCRIPTION
**Pull request by aHenryJard**
Mainly to add the purpose of this plugin and how to use it.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by AnEmortalKid — APPROVED** [review]

**Comment by AnEmortalKid** [comment]
@jglick any chance this can be merged easily?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Jenkins UI sample plugin
+
+This plugin helps Jenkins developpers to build user interface using Jenkins UI components.
```
Comment (jglick) [inline README.md:3]:
```suggestion
This fake plugin shows Jenkins developers how to build user interfaces using Jenkins UI components.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+This plugin contains:
+- demonstration of UI controls available in Jenkins based on Stapler, Jelly, Groovy, etc.
+- snippets of jelly/groovy "ready-to-use" examples
```
Comment (jglick) [inline README.md:7]:
```suggestion
- snippets of Jelly/Groovy "ready-to-use" examples
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Usage
+
+Run `mvn hpi:run`
+
+Go on "UI Samples" menu item or at http://localhost:8080/jenkins/ui-samples/ and play with UI components :).
```
Comment (jglick) [inline README.md:13]:
```suggestion
Go to the **UI Samples** menu item or straight to http://localhost:8080/jenkins/ui-samples/ and play with UI components.
```

**Pull request merged**